quora/quora.py

```diff
@@ -122,10 +122,10 @@ def scrape_question_stats(soup):
 
         question_dict = {'want_answers' : try_cast_int(want_answers),
                          'answer_count' : try_cast_int(answer_count),
-                         'question_text' : question_text,
+                         'question_text' : question_text.string,
                          'topics' : topics,
-                         'question_details' : question_details,
-                         'answer_wiki' : str(answer_wiki),
+                         'question_details' : question_details.string,
+                         'answer_wiki' : answer_wiki.string,
                         }
```

Comment:
.string will cause an error if there is a link or any other tag. (Or ignore it and turn stuff into none apparently..)
```python
print Quora.get_question_stats('What-are-the-best-Cyanide-Happiness-comics')
{'want_answers': 2, 'question_text': u'What are the best Cyanide & Happiness comics?', 'topics': [u'Communication', u'Writing', u'Books', u'Publishing', u'Comics (narrative art form)'], 'question_details': None, 'answer_count': 474, 'answer_wiki': None}
```
question_details should've been:
```
<div class="question_details_text inline_editor_content"><span class="qlink_container"><a class="external_link" href="http://www.explosm.net/" rel="nofollow" target="_blank">http://www.explosm.net/</a></span><br/><br/><span class="qlink_container"><a class="external_link" href="http://en.wikipedia.org/wiki/Cyanide_and_Happiness" target="_blank">http://en.wikipedia.org/w<wbr></wbr>iki/Cya...</a></span></div>
```